pr-data.csv

```diff
@@ -1396,125 +1396,129 @@ https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modu
-https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.VolatilePageMemoryPartitionStorageTest.testInsertAll,ID,MovedOrRenamed,,
-https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.VolatilePageMemoryPartitionStorageTest.testInsertClosure,ID,MovedOrRenamed,,
-https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.VolatilePageMemoryPartitionStorageTest.testInsertClosure_failureBranch,ID,MovedOrRenamed,,
-https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.VolatilePageMemoryPartitionStorageTest.testReadAll,ID,MovedOrRenamed,,
-https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.VolatilePageMemoryPartitionStorageTest.testRemoveAll,ID,MovedOrRenamed,,
-https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.VolatilePageMemoryPartitionStorageTest.testRemoveAllExact,ID,MovedOrRenamed,,
-https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.VolatilePageMemoryPartitionStorageTest.testRemoveAllExact_failureBranch,ID,MovedOrRenamed,,
-https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.VolatilePageMemoryPartitionStorageTest.testRemoveAllKeyNotExists,ID,MovedOrRenamed,,
-https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.VolatilePageMemoryPartitionStorageTest.testReplaceExactClosure,ID,MovedOrRenamed,,
-https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.VolatilePageMemoryPartitionStorageTest.testReplaceExactClosure_failureBranch,ID,MovedOrRenamed,,
-https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.VolatilePageMemoryPartitionStorageTest.testWriteAll,ID,MovedOrRenamed,,
+https://github.com/apache/ignite-3,747be38e757f39b310879a6158262d4414020ed6,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.mv.AbstractPageMemoryMvPartitionStorageTest.committedMultiPageValuesAreReadSuccessfully,ID,,,
```

Comment:
The new test name can't be that as this class is abstract: https://github.com/apache/ignite-3/blob/88c5fa34012b29184fc388c5fd30bef9bc4b5a82/modules/storage-page-memory/src/test/java/org/apache/ignite/internal/storage/pagememory/mv/AbstractPageMemoryMvPartitionStorageTest.java#L34

Reply:
Thanks for noticing! The NonDex run result is "No test found".
But this is the old test file changed to. 
How should I fix it, just remove the new rows and change to "Deleted" as others ?

Reply:
Don't run `mvn -Dtest=AbstractPageMemoryMvPartitionStorageTest#committedMultiPageValuesAreReadSuccessfully ...` (either `test` or `nondex:nondex`). A test with that name doesn't exist. You can either (1) check what concrete subclasses of `AbstractPageMemoryMvPartitionStorageTest` exist and then run the test method for those classes (or even simpler just run all test methods in those classes), or (2) run all tests in the module (e.g., just `mvn nondex:nondex`) and see what test names get generated.

Reply:
I think you should run NonDex on the whole project to find if any new flaky tests are related to the renaming

Reply:
HI,
I deleted the four new rows, ran all tests in the module, and found some new tests and the "renamed" test that we discussed.
But I plan to open another PR for these new tests.
The others should be correct since their file are deleted.
So, it's there any problem with this PR?